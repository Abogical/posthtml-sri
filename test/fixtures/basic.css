html {
	display: flex;
	height: 100%;
}

body {
	display: flex;
}
