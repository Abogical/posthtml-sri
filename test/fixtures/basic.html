<!DOCTYPE html>
<html>
	<head>
		<script src="basic.js"></script>
		<link rel="stylesheet" href="basic.css" />
	</head>
</html>
