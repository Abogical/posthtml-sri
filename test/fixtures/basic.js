const a = 9;
const b = 8;
alert(a + b);
