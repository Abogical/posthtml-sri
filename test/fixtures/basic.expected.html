<!DOCTYPE html>
<html>
	<head>
		<script src="basic.js" integrity="sha512-MUS7Gdcgr3In98ODhITyPjXWdWE9dezoOcyopcyYXzpFz2LrOApCRupkTwgiaS31+DhQacgbQ5T1hlGWPiBNGQ=="></script>
		<link rel="stylesheet" href="basic.css" integrity="sha512-CVazCeMWuNKPH6VzeuYxVGC3JKXVQB/wqncMrpHoS3wJclPNeKN+SxO/zz1A9U3s2zczk/Yx1iZEJ6+CwJW81A==">
	</head>
</html>
